setup.py

```diff
@@ -2,7 +2,7 @@
 import os
 
 
-gammapy = "gammapy~=0.18"
+gammapy = "gammapy~=0.18.0"
```

Comment:
Not a problem, but what's the need for this variable substitution?

Reply:
We have `gammapy` in three optional dependencies. This is to make sure, we use the same version everywhere. I just forgot it at one point (used the variable in 2 and the string in another one)